.github/workflows/build-test.yml

```diff
@@ -41,5 +41,5 @@ jobs:
 
       - name: Race Condition Tests
         if: ${{ matrix.os != 'windows-latest' }} # known issue: https://github.com/golang/go/issues/46099
-        run: go run -race . -u scanme.sh
+        run: go run -race . -l ../race-condition-test/targets.txt -tags xss,dns
```

Comment:
Please move `targets.txt`, for example, into `/v2/cmd/functional-test/targets.txt` since it pertains to runtime testing